NetCoreRestApi/DataLayer.EF/DbSetMock.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace DataLayer.EF
+{
+    public class DbSetMock<TEntity> : DbSet<TEntity>
```

Comment:
As temporary solution, it is ok. later this will be replaced with EF core dbcontext in memory

Reply:
ok